### PR TITLE
Add timeout option

### PR DESCRIPTION
**Pull request by masahiro331**
Add timeout option.

issue: #139 

note: 
https://github.com/aquasecurity/fanal/blob/master/analyzer/analyzer.go#L115
It is left as a default value of the fanal library.

### CODE REVIEW

File: pkg/run.go [hunk 1]
```diff
@@ -30,6 +30,8 @@ func Run(c *cli.Context) (err error) {
 		l.Fatal(err)
 	}
 
+	os.Setenv("TRIVY_TIMEOUT_SEC", c.String("timeout"))
```
Comment (knqyf263) [inline pkg/run.go]: You should not use environment variable. It's better to pass it below:
https://github.com/aquasecurity/trivy/blob/master/pkg/scanner/scan.go#L28

Comment (masahiro331) [reply pkg/run.go]: Add dockerOption argument for ScanImage?

Comment (knqyf263) [reply pkg/run.go]: Add timeout to `scanOptions`

Comment (masahiro331) [reply pkg/run.go]: fixed

File: cmd/trivy/main.go [hunk 2]
```diff
@@ -117,6 +117,11 @@ OPTIONS:
 			Value: utils.DefaultCacheDir(),
 			Usage: "cache directory",
 		},
+		cli.IntFlag{
```
Comment (tomoyamachi) [inline cmd/trivy/main.go]: You can set DurationFlag.

Comment (knqyf263) [reply cmd/trivy/main.go]: @masahiro331 Could you fix it?

Comment (masahiro331) [reply cmd/trivy/main.go]: I will fix it.
Please wait a few days.

Comment (masahiro331) [reply cmd/trivy/main.go]: Fixed

File: pkg/scanner/scan.go [hunk 3]
```diff
@@ -29,6 +30,9 @@ func ScanImage(imageName, filePath string, scanOptions types.ScanOptions) (map[s
 		if err != nil {
 			return nil, xerrors.Errorf("failed to get docker option: %w", err)
 		}
+		if scanOptions.Timeout != time.Duration(60*time.Second) {
```
Comment (knqyf263) [inline pkg/scanner/scan.go]: You don't have to check for equality.

**Comment by knqyf263** [comment]
@masahiro331 Thanks.

**Pull request merged**